### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Nizamuddin4493**
# Math magicians: components 🎉  ✔️ 🎉 ✔️ 
- Used React components.
- Used React props.
- Used React state.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './component/Calculator';
 
 function App() {
```
Comment (topeogunleye) [inline src/App.js]: - [ ] As required by the project requirements, I kindly suggest that you make `App.js` component class-based. In future Milestones, you'll convert it back to a function based component.

## EXAMPLE
```
class App extends React.Component {
  
}
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Nizamuddin4493, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
>- [ ] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Bad Commit Messages
image
> ### Good Commit Messages:
>- [x]  Add fix for dark mode toggle state
>- [x] Update README file.
>- [x] Add styles to h1.
>
>- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your @topeogunleye so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,17 @@
+import React, { Component } from 'react';
 import Calculator from './component/Calculator';
 
-function App() {
-  return (
-    <Calculator />
-  );
+class App extends Component {
```
Comment (mateo951) [inline src/App.js:4]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
Hi @Nizamuddin4493, 

Well done making the changes requested by the previous reviewer 🤓
Your project is complete!  There is nothing else to say other than... it's time to merge it 🙌 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**